Geometry/RPCGeometryBuilder/python/rpcGeometry_cfi.py

```diff
@@ -1,10 +1,5 @@
-import FWCore.ParameterSet.Config as cms
-
-#
-# This cfi should be included to build the RPC geometry model.
-#
-RPCGeometryESModule = cms.ESProducer("RPCGeometryESModule",
-    useDDD = cms.untracked.bool(True)
-)
+from Geometry.RPCGeometryBuilder.RPCGeometryESModule_cfi import RPCGeometryESModule
 
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
 
+dd4hep.toModify(RPCGeometryESModule, useDDD = False, useDD4hep = True)
```

Comment:
Configuration objects should be modified only in the files that define them, i.e. the `RPCGeometryESModule` imported from `Geometry.RPCGeometryBuilder.RPCGeometryESModule_cfi` should be cloned here.

Reply:
But cloning will change the name of the process which I want to avoid. rpcGeometry_cfi.py is included in all places and everywhere the name of the process is required to be RPCGeometryESModule. Please suggest how this can be achieved

Reply:
Minimally
```python
from Geometry.RPCGeometryBuilder.RPCGeometryESModule_cfi import RPCGeometryESModule as _RPCGeometryESModule
RPCGeometryESModule = _RPCGeometryESModule.clone()
```

A less confusing way is recommended in coding rules 6.14 by using
```cpp
  descriptions.add("RPCGeometryESModuleDefault", desc);
```
in the `fillDescriptions()` and then here
```python
from Geometry.RPCGeometryBuilder.RPCGeometryESModuleDefault_cfi import RPCGeometryESModuleDefault as _RPCGeometryESModuleDefault
RPCGeometryESModule = _RPCGeometryESModuleDefault.clone()
```

(I also want to note that the code rules state that the module label should match the `cfi` file name, which is not the case here, but this is certainly beyond this PR)

Reply:
```python
from Geometry.RPCGeometryBuilder.RPCGeometryESModule_cfi import RPCGeometryESModule as RPCGeometryESModuleDefault
RPCGeometryESModule = RPCGeometryESModuleDefault.clone()
```

Reply:
> ```python
> from Geometry.RPCGeometryBuilder.RPCGeometryESModule_cfi import RPCGeometryESModule as RPCGeometryESModuleDefault
> RPCGeometryESModule = RPCGeometryESModuleDefault.clone()
> ```

This would effectively add `RPCGeometryESModuleDefault` to the Process as well.

Reply:
That's true, importing with a leading underscore is better.

Reply:
@bsunanda Is this issue resolved?